.github/run_browserstack_acceptance.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
```

Comment:
nit but what do you think about removing this space?

Reply:
sure